content/kotlin/concepts/strings/terms/lowercase/lowercase.md

```diff
@@ -0,0 +1,54 @@
+---
+Title: '.lowercase()'
+Description: 'The lowercase method in Kotlin is used to convert the characters of a string to lowercase.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Lowercase'
+  - 'Strings'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The `lowercase()` method in Kotlin is used to convert the characters of a string to lowercase. 
+
+## Syntax
+
+```pseudo
+fun String.lowercase(locale: Locale = Locale.getDefault()): String
+```
+
+### Parameters
+
+- `locale` (optional): A `Locale` object representing the language and region-specific rules for case conversion. The default value is `Locale.getDefault()`.
+
+## Examples
+Let's dive into some examples to see how the `.lowercase()` method can be used:
+
+
+```
+println("Iced frappé!".lowercase()) // iced frappé!
+```
+
+The output of this code will be:
+
+```shell
+hello, world!
+```
+
+Let's take a look at an example where we specify the locale:
+
+```pseudo
+println("KOTLIN".lowercase()) 
+val turkishLocale = Locale.forLanguageTag("tr")
+println("KOTLIN".lowercase(turkishLocale))
+```
```

Comment:
```suggestion
```kotlin
import java.util.Locale

fun main() {
	println("KOTLIN".lowercase()) 
	val turkishLocale = Locale.forLanguageTag("tr")
	println("KOTLIN".lowercase(turkishLocale)) 
}

```
```
To work with locales in Kotlin, you should use the java.util.Locale class, which is a part of the Java standard library.